### PR TITLE
Uncertainty defaults fix

### PR DESCRIPTION
**Pull request by mjprilliman**
## Description

-Fixes from #1812 result in calculation error. Updating to non-zero std. dev. to avoid error for default case

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change modifies variables in existing compute modules. Please see [Checking for PySAM Incompatible API Changes](https://github.com/NREL/SAM/wiki/Checking-for-PySAM-Incompatible-API-Changes).

## Checklist:

If you have added a new compute module in a SSC pull request related to this one, be sure to check the [Process Requirements](https://github.com/NREL/SAM/wiki/Compute-modules-in-SAM).

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/pvuncertainty.cpp [hunk 1]
```diff
@@ -114,13 +114,13 @@ PVUncertaintyForm::PVUncertaintyForm( wxWindow *parent, Case *cc )
...
-	sourceinfo.push_back(std::make_tuple("Spectral response", "Uncertainty in differences between the reference spectral shape of incident irradiance at STC during\nindoor tests for module rating and outdoor operating conditions.", "Spectral response:1:0:0:0:0"));
-	sourceinfo.push_back(std::make_tuple("Cell temperature", "Uncertainty in measurement of the module temperature coefficient of power. Consider the parameters on the Module page.", "Cell temperature:1:0:0:0:0"));
+	sourceinfo.push_back(std::make_tuple("Spectral response", "Uncertainty in differences between the reference spectral shape of incident irradiance at STC during\nindoor tests for module rating and outdoor operating conditions.", "Spectral response:1:0:0.1:0:0"));
+	sourceinfo.push_back(std::make_tuple("Cell temperature", "Uncertainty in measurement of the module temperature coefficient of power. Consider the parameters on the Module page.", "Cell temperature:1:0:0.1:0:0"));
```
Comment (sjanzou) [inline src/pvuncertainty.cpp:123]: Thanks for updating these - at some point we should revisit moving them to the defaults.

**Review by sjanzou — APPROVED** [review]
Thanks for updating!

**Comment by mjprilliman** [comment]
@janinefreeman I realized my last fix on this cause the defaults to error. I can make further changes on defaults per our conversation yesterday, or leave it to the next release to do some more investigation into it.

**Review by janinefreeman — APPROVED** [review]

**Comment by janinefreeman** [comment]
let's go with these for now, and see what kind of feedback we get, or if we hear of any additional data. Thanks!

**Pull request merged**